web/models.py

```
@@ -2804,3 +2806,482 @@ class ScheduledPost(models.Model):
+        recipient_list = [self.user.email]
+        try:
+            send_mail(subject, message, from_email, recipient_list, fail_silently=True)
+        except SMTPException:
+            logger = logging.getLogger(__name__)
+            logger.exception("Error sending application status email")
+            
+    def _create_membership_if_approved(self) -> None:
+        """Create membership for approved applications."""
+        if self.status == 'approved':
+            # Check if membership already exists first
+            if not ChapterMembership.objects.filter(user=self.user, chapter=self.chapter).exists():
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)